Makefile

```diff
@@ -49,3 +49,28 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+
+.PHONY: bench
+bench:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee bench/bench.txt
+	cd bench && git clone https://go.googlesource.com/perf && cd perf && go install ./cmd/benchstat
```

Comment:
It should be a separate target, otherwise you will install benchstat on every test run

Reply:
Fixed.